en/README.md

```diff
@@ -0,0 +1,1142 @@
+    "retryIntervalSec": 10
+  },
+  "jsonrpc": "2.0"
+}
+```
+
+## Tables
+
+### armInfoStatus
+
+|Status|Description|
+|:---------|:---|
+|"INIT"   |Initial status. This status represents that not communication yet.|
+|"OK"     |Succeed connection.|
+|"NG"     |Fail connection.|
+
+### ServerType
+
+If you create a HAP, it needs to define new server type which has no existence UUID as below table.
+
+|Name|UUID|
+|:---|:---|
+|Zabbix    |8e632c14-d1f7-11e4-8350-d43d7e3146fb|
+|Nagios    |902d955c-d1f7-11e4-80f9-d43d7e3146fb|
+|Ceilometer|aa25a332-d1f7-11e4-80b4-d43d7e3146fb|
+
+### triggerSeverity
+
+|Severity|Description|
+|:---|:--|
+|"UNKNOWN"  |Unknown|
+|"INFO"     |Information|
+|"WARNING"  |Warning|
+|"ERROR"    |Error|
+|"CRITICAL" |Critical|
+|"EMERGENCY"|Emergency|
+
+### triggerStatus
+
+|Status|Description|
+|:---------|:---|
+|"OK""    |Succeed connection.|
+|"NG"     |Fail connection.|
+|"UNKNOWN"|Status is unknown|
+
+### updateType
+
+|Type|Description|
+|:---------|:---|
+|"ALL"    |Send all data. Remove old data from hatohol server and regist all sent data.|
+|"UPDATED"|Send only updated data. After that if data ID(s) already exist in Hatohol server, they will be overwritten. Or, if they do not exist in Hatohol Server, they are newly registered in Hatohol server.|
+
+### eventType
+
+|Type|Description|
+|:---|:---------:|
+|"GOOD"        |Normal|
+|"BAD"         |Abnormal|
+|"UNKNOWN"     |Unknown|
+|"NOTIFICATION"|Notification|
+
+### putResult
+
+The following table values represent whether success or failure when calling put procedure series. If it fails to update database, recalling procedure is standard behavior.
+
+|Status|Description|
+|:---------|:---|
+|"SUCCESS"|Update is succeed.|
+|"FAILURE" |Update is fail.|
+
+If put series of procedures' parameters are wrong or invalid and so on, it needs to return error which is defined in the JSON RPC 2.0 to HAP as following example:
+
+ ```json
+{
+  "id": "1",
+  "error": {
+    "message": "Invalid params",
+    "code": -32602
+  },
+  "jsonrpc": "2.0"
+}
+ ```
+
+### fetchResult
+
+The following table values represent whether success or failure when calling fetch series of procedures. If a request fails, resending request to plugin is standard behavior.
+
+|Status|Description|
+|:---------|:---|
+|"SUCCESS"|Succeed to accept a request.|
+|"ABBREV" |Abbreviate to accept a request because request interval a little.| 
+|"FAILURE"|Fail to accept a request.|
+
+If fetch series of procedures' parameters are wrong or invalid and so on, it needs to return error which is defined in the JSON RPC 2.0 to HAP as following example:
+ 
+ ```json
+{
+  "id": "1",
+  "error": {
+    "message": "Invalid params",
+    "code": -32602
+  },
+  "jsonrpc": "2.0"
+}
+ ```
+
+
+<!--
+## Revision history
+-->
+
+## Getting help 
+If you find a bug in this documentation or would like to propose an improvement, please send a bug report to Hatohol community mailing list. [hatohol-users@sourceforge.net]
```

Comment:
sourceforge.jp is migrated to osdn,jp.
I would be great to modify this address like this: hatohol-users@osdn.jp
Or we continue to use this mailing list address?